src/mlpack/methods/reinforcement_learning/environment/env_type.hpp

```diff
@@ -106,6 +106,94 @@ class DiscreteActionEnv
 size_t DiscreteActionEnv::State::dimension = 0;
 size_t DiscreteActionEnv::Action::size = 0;
 
+/**
+ * To use the dummy environment, one may start by specifying the state and
+ * action dimensions.
+ * Eg:
+ *  ContinuousActionEnv::State::dimension = 3;
+ *  ContinuousActionEnv::Action::size = 1;
```

Comment:
Let's use `@code` here is an example:

https://github.com/mlpack/mlpack/blob/2326bf8bd0b380c17eed6b5b1c34fbfab1563774/doc/policies/functiontype.hpp#L43-L48